src/main/scala/system/Generator.scala

```diff
@@ -82,7 +80,26 @@ object Generator extends GeneratorApp {
       TestGeneration.addSuites(rvi.map(_("p")))
       TestGeneration.addSuites((if (vm) List("v") else List()).flatMap(env => rvu.map(_(env))))
       TestGeneration.addSuite(benchmarks)
-      TestGeneration.addSuite(new RegressionTestSuite(if (xlen == 64) rv64RegrTestNames else rv32RegrTestNames))
+
+      /* Filter the regression tests based on what the Rocket Chip configuration supports */
+      val extensions = {
+        val fd = coreParams.fpu.map {
+          case cfg if cfg.fLen >= 64 => "d"
```

Comment:
should this be "fd"?

Reply:
Yeah, this should be... Fixed in the force-push.